java/README.md

```diff
@@ -0,0 +1,26 @@
+## Pack64 Java
+
+### To compile
+
+javac src/com/luminoso/pack64/*.java
+
+### To run unit tests
```

Comment:
These instructions sound to me like there's a build system for Java that we should be using, but aren't.

Reply:
Just the command line Java commands to build and run the unit tests. It would just save someone some effort of figuring out the compile commands if they download and want to test locally. Something like Maven (a pom.xml file) could be added, but would seem overkill for a single Java file.  We don't have this issue with python/Javascript because they are interpreted.  Though I can create a Maven build file if you would like that? As I write all that it might make sense to add the Maven build to create a pack64.jar file which would be an added benefit and easier integration point.

Reply:
Okay, I added the Maven build files to simplify the build and testing. My first conversation in my head lead me not to use a build system at all and seemed correct. The later conversations in my head had me wondering about my earlier choices. All these conversations in my head have me questioning my sanity. Anyway. Good call on adding the build system.